binding.gyp

```diff
@@ -18,7 +18,7 @@
         [ "OS == 'mac'", {
           "libraries": [
             "../deltachat-core/builddir/src/libdeltachat.a",
-            "../deltachat-core/builddir/libs/libetpan/libetpan.a",
+            "/usr/local/Cellar/libetpan/1.9.2_1/lib/libetpan.a",
```

Comment:
I hope this will be the same on each machine. We might want/need to parametrize it somehow later.

Reply:
We should also be able to use `/usr/local/lib/libetpan.a` will give it a try

Reply:
But maybe it's even more clean to use this `libetpan-config` tool.

Reply:
Maybe yeah, but lets get it to work first :wink: